Wifi off (#777)
Xilinx/PYNQ: docs/source/faqs.rst

to the wireless network. Your host PC can then connect to the same wireless
network to connect to the board.

How can I enable wireless access point?
---------------------------------------

> If the board has a built-in wireless module, and the PYNQ image has the 

Reviewer: ```suggestion
If the board has a supported built-in wireless module, and the PYNQ image has the 
```

> *wpa_ap* package installed, you can probably start the WiFi access point.

Reviewer: ```suggestion
*wpa_ap* package installed, you can try start the WiFi access point as follows:
```

Does Pynq support Python 2.7?
-----------------------------

> Python 2.7 is loaded on Zynq and Zynq Ultrascale; Python 2.7 scripts can be executed. 

Reviewer: Clarify this FAQ? ```suggestion
The PYNQ image is based on Ubuntu which includes Python 2.7.
Python 2.7 can be used on the board, but is not supported by PYNQ. 
Depending on the PYNQ image version you are using, you will see other Python versions are also installed.
```

> The PYNQ v2.0 and later releases are based on Python 3.6 and are not backward compatibility 

Reviewer: ```suggestion
The PYNQ v2.0 and later releases are based on Python 3.6 and are not backward compatible 
```
